blocks/form/wizard.js

```diff
@@ -0,0 +1,50 @@
+function createButton(fd) {
+  const button = document.createElement('button');
+  button.type = 'button';
+  button.textContent = fd.Label;
+  button.className = 'form-wizard-step-button';
+  return button;
+}
+
+const createTooltip = (() => {
+  const tooltip = document.createElement('div');
+  tooltip.className = 'field-tooltip';
+  document.addEventListener('click', () => tooltip.remove(), true); // execute for next click
+  return (fd) => {
+    tooltip.textContent = fd.Label;
+    return tooltip;
+  };
+})();
+
+function moveToNext(current) {
+  const invalid = current.querySelector(':invalid');
+  if (invalid) {
+    const tooltip = createTooltip({ Label: invalid.validationMessage });
+    invalid.parentElement.insertBefore(tooltip, invalid.nextElementSibling);
+  } else {
+    current.classList.remove('current-wizard-step');
+    current.nextElementSibling.classList.add('current-wizard-step');
+  }
+}
+
+function moveToPrev(current) {
+  current.classList.remove('current-wizard-step');
+  current.previousElementSibling.classList.add('current-wizard-step');
+}
+
+export default async function decorate(form) {
+  form.classList.add('form-layout-wizard');
+  [...form.children].forEach((step) => {
+    const current = form.getElementsByClassName('current-wizard-step');
```

Comment:
for consistency please use `querySelector`

Reply:
`getElementsByClassName` gives live HTML collections, so I don't need to use `querySelector` to find the current step every time I move back or next step.